### PR TITLE
Creation of influenza-bioinfo-solutions.md

### PR DESCRIPTION
**Pull request by cinnetcrash**
Influenza guidance doc has converted to markdown and uploaded to the docs section. 
Images will be added shortly.

### CODE REVIEW

File: docs/influenza-bioinfo-solutions.md [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+one large open reading frame (ORF). These genomic structures are
+crucial for the virus's ability to use host cellular machinery for
+replication and protein synthesis. The viral RNAs are incorporated
+into viral ribonucleoproteins (vRNPs), which consist of multiple
+copies of nucleoprotein (NP) and RNA-dependent RNA polymerase complex (RdRp), composed of PB1, PB2, and PA subunits. This assembly
+facilitates both the transcription of vRNA into viral mRNA, complete
+with a 5′ cap and a 3′ poly(A) tail, and the replication of vRNA
+through a complementary RNA (cRNA) intermediate. M segment of the
+virus coding matrix gene.**  
+**<img src="images/influenza-guidance-fig1.png" style="width:3.95313in;height:3.28467in" />
```
Comment (frankambrosio3) [inline docs/influenza-bioinfo-solutions.md:40]: This image is not rendering in my preview tab. Needs tags like this:

<p align="center">
  
</p>

**Review by frankambrosio3 — APPROVED** [review]

File: docs/influenza-bioinfo-solutions.md (hunk 1 above)
Comment (tralynca) [reply docs/influenza-bioinfo-solutions.md:40]: Mine rendered, but I had to change the link. The path started with media instead of images.

Comment (tralynca) [reply docs/influenza-bioinfo-solutions.md:40]: Ah, the paths were renamed, but as you said, the images were also not added

**Comment by emily-smith1** [comment]
Please wait to merge this. There are missing citations, figure numbers, and formatting inconsistencies that need to be addressed. The readme also needs to be updated.

**Comment by tralynca** [comment]
Hey Emily,Yes, I explained to Gultekin that it will cause too much confusion for me to also work on this. Frank still had the doc on his patch branch anyway, so submitting to the main branch won’t solve anything.So, I’ll wait for him to resolve the issues on his end.Thanks for the fair warning though. On 08 May 2024, at 21:16, Emily Smith ***@***.***> wrote:﻿
Please wait to merge this. There are missing citations, figure numbers, and formatting inconsistencies that need to be addressed. The readme also needs to be updated.

**Comment by tralynca** [comment]
will create a new PR from dev branch and linking the correct issue

**Pull request closed without merging**